### ISSUE TITLE
Feature request: A not-asked Question to be able to be displayed 

### ISSUE THREAD

**Issue by GHNewbiee** [SBoudrias/Inquirer.js#1076]
**The case**
A wish to display questions (for reference) which are not asked through `when` property.

**The problem**
`when` property just hides the question when its value is `false`.

**Potential solution**
- A new property, or
- an extra parameter to `when` property function

which will declare if a not-asked (not-activated) question should be displayed or not.

**Example**
```js
import inquirer from "inquirer";

inquirer
  .prompt([
    {
      type: 'confirm',
      name: 'fruit',
      message: 'Is it a fruit?',
      default: true
    },
    {
      type: 'confirm',
      name: 'sweet',
      message: 'Is it sweet?',
      default: true,
      when: function (answers, status) {
        status.activated = answers.fruit ? false : true;
        status.displayed = true;
        return status; 
      },
    },
    {
      type: 'confirm',
      name: 'example',
      message: 'Is it a good example?',
      default: true,
    }
  ])
  .then(() => {
    ...
  })
```

```
? Is it a fruit? Yes
? Is it sweet? Yes (# default value and, for example, in very light colour to imply that it's not activated)
? is it a good example? Y/n
...
```

*Tia*

**Comment by SBoudrias** [comment]
Yeah, I think you'd want to wrap the question in `colors.dim(question)`.

Besides that, I think you'll find it a little hard to properly implement everything; but happy if you want to take a stab and we can work out of a PR.

We'll need to do the question line rendering inside `inquirer` which currently has no rendering logic; and so for that you'll need to make sure the rendering respect the `theme` options.

As far as interface go, I don't have a strong opinion. IMO allowing to return an object from `when` isn't a bad idea. But let's not provide an interface requiring mutation - and make sure you maintain backward compatibility. Something like `when(answers: Answers): boolean | { display: boolean; ask: boolean; }`
